docs/core/extensions/snippets/configuration/console-di-ienumerable/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
```

Comment:
access-by-line.txt?

Reply:
Good callout, I am adding it for all the new extensions/fundamentals content - thanks.